tests/testthat/test-formatters.R

```diff
@@ -925,3 +925,24 @@ test_that("fmt_config works as expected", {
   expect_silent(obj_na_str(x) <- "something wrong")
   expect_silent(obj_align(x) <- "something wrong")
 })
+
+# regression tests -------------------------------------------------------------
+test_that("na_str works when having multiple NAs and multiple values to be changed", {
+  # Regression test from #308
+  expect_identical(
+    format_value(c(1, 1, NA), format = "xx.x (xx.x - xx.x)", na_str = "NE"),
+    "1.0 (1.0 - NE)"
+  )
+  expect_identical(
+    format_value(c(1, NA, NA), format = "xx.x (xx.x - xx.x)", na_str = "NE"),
```

Comment:
This was messy before. Recycling did not work

Reply:
hi davide, sorry, what recycling

Reply:
vector recycling did not work when na_str was length one